### PR TITLE
[PR on PR 617] Pt 154246728 introduce mnesia 2 - update doc

### PR DESCRIPTION
**Pull request by lucafavatella**

### CODE REVIEW

File: docs/docker-testnet.md [hunk 1]
```diff
@@ -66,12 +66,12 @@ docker run -d -p 3013:3013 -e PEERS_ADDRESS_0=http://31.13.248.103:3013/ aetrnty
 
 ## Persisting Data
 
-To persist blockchain data and node keys between container runs, use [Docker volumes](https://docs.docker.com/engine/admin/volumes/volumes/). Replace `~/.aeternity/.chain` with location of your choice.
+To persist blockchain data and node keys between container runs, use [Docker volumes](https://docs.docker.com/engine/admin/volumes/volumes/). Replace `~/.aeternity/data` with location of your choice.
```
Comment (dincho) [inline docs/docker-testnet.md]: Is the default db_path = `data` now ?
If so, IMO we should also change the example configuration files

Comment (lucafavatella) [reply docs/docker-testnet.md]: Do you mean https://github.com/aeternity/epoch/blob/999c46392991310c8dfcfda83f8fda152f5e537e/apps/aeutils/test/data/epoch_full.yaml#L43 ?

Comment (uwiger) [reply docs/docker-testnet.md]: Not sure we have to change anything here. The mnesia database will place itself in whatever directory we want, and will coexist there with the old persistence files.

Comment (lucafavatella) [reply docs/docker-testnet.md]: > Not sure we have to change anything here.

@uwiger  For sure a constraint we have in `-v ~/.aeternity/data:/home/epoch/node/data` is that *with the config that the Docker image uses* the DB dir must be `/home/epoch/node/data`.

**Comment by lucafavatella** [comment]
> The PR needs to be rebased ...

@uwiger Rebased.

File: docs/docker-testnet.md (hunk 1 above)
Comment (lucafavatella) [reply docs/docker-testnet.md]: Linked this from https://www.pivotaltracker.com/story/show/154636528/comments/185105839

**Comment by dincho** [comment]
Last time I checked data dir contained some other file as well. We're using docker bind mounts and this information will be obfuscated see https://docs.docker.com/engine/admin/volumes/

**Comment by uwiger** [comment]
@dincho to be clear, mnesia will create a "mnesia" subdirectory under the DbPath, and write nothing outside of it.

**Comment by dincho** [comment]
@uwiger I shouldn’t post a general comment but to specific line. I’m concerned the Docker example changes in this PR

**Comment by lucafavatella** [comment]
This PR is now invalid as the target branch / PR is now merged. So I shall close this and reopen against master.

**Comment by lucafavatella** [comment]
See #632.

**Pull request closed without merging**